build-utils/sentry-instrumentation.js

```diff
@@ -93,10 +92,8 @@ class SentryInstrumentation {
                 size: `${Math.round(sizeInKb)} KB`,
               },
             });
-            span.startTimestamp = start;
-            span.finish();
-            span.timestamp = start + sizeInKb / 1000;
-            transaction.finish(true);
+            span.finish(start + sizeInKb / 1000);
```

Comment:
@billyvg I made the changes to the file to use the new API.
But this line here in terms of calculating the `endtimestamp` for the Span seems very odd.
Time + Filesize? What is the reason behind this?
I suppose you are abusing the Span chart to show file size?

Reply:
Correct